### PR TITLE
Add missing dependencies to setup.cfg

### LINKED ISSUE #291: Missing dependancies
We import `aiohttp` and `tqdm` directly but only rely on them via transitive dependencies.

### PR DESCRIPTION
**Pull request by SolarDrew**
Fixes #291

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -36,6 +36,8 @@ install_requires =
   sunpy[net,asdf]>=4.0.7
   setuptools>=59
   jsonschema>=3.2
+  aiohttp>=3.6
```
Comment (Cadair) [inline setup.cfg:39]: The fact we are doing this for typehints is somewhat annoying but hey.

**Comment by Cadair** [comment]
@SolarDrew The oldestdeps test fail is a deprecation warning from Python, we need to ignore it in the `setup.cfg` file to make the build pass.

**Comment by SolarDrew** [comment]
Would it not be better to remove the deprecated feature and drop 3.9, or are we not ready to do that yet?

**Comment by Cadair** [comment]
The issue is that the oldest version of aiohttp uses the feature deprecated in 3.8

File: setup.cfg (hunk 1 above)
Comment (SolarDrew) [reply setup.cfg:39]: We are? I was just going by NEP 29.

**Pull request merged**